src/OpenCL/Extensions/Silk.NET.OpenCL.Extensions.ARM/ArmImportMemory.gen.cs

```diff
@@ -4,12 +4,14 @@
 // of the MIT license. See the LICENSE file for details.
 using System;
 using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
 using System.Text;
 using Silk.NET.OpenCL;
 using Silk.NET.Core.Native;
 using Silk.NET.Core.Attributes;
 using Silk.NET.Core.Contexts;
 using Silk.NET.Core.Loader;
+using Extension = Silk.NET.Core.Attributes.ExtensionAttribute;
```

Comment:
Generated file but this ease-of-use feature?

Reply:
This is actually to resolve a conflict as there's a ExtensionAttribute somewhere in the BCL of .NET Standard 2.1

Reply:
Fair enough, `System.Runtime.CompilerServices` thought not sure why not just use `Silk.NET.Core.Attributes.ExtensionAttribute` directly

Reply:
Meh could do, just a matter of personal preference really. Can change if you require.

Reply:
no, ofc not. just a nit